<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupForm</class>
 <widget class="QDialog" name="CreateGroupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 2px solidrgb(45, 45, 45);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="Main_frame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background-color: rgb(30, 30, 30);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="Title_bar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="Title">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="Title_label">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 20px;</string>
              </property>
              <property name="text">
               <string>Добавление группы</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Button">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border: none;
	border-radius: 8px;
	background-color: rgb(230, 0, 0);
}

QPushButton:hover
{
	border: none;
	border-radius: 8px;
	background-color: rgb(200, 0, 0);
}

QPushButton:pressed
{
	border: none;
	border-radius: 8px;
	background-color: rgb(160, 0, 0);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Content_bar">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>57</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>251</width>
             <height>86</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>251</width>
             <height>86</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="groupNameEdit">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	border: 2px solid rgb(38, 38, 38);
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	font-family: 'Roboto Regular';
	font-size: 16px;
	background-color: rgb(35, 35 , 35);
}

QLineEdit:hover
{
	border: 2px solid rgb(48, 48, 48);
}
QLineEdit:focus
{
	border: 2px solid rgb(0, 180, 255);
	background-color: rgb(45, 45, 45);
}
</string>
              </property>
              <property name="placeholderText">
               <string>Название группы</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="numberOfUsersEdit">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	border: 2px solid rgb(38, 38, 38);
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	font-family: 'Roboto Regular';
	font-size: 16px;
	background-color: rgb(35, 35 , 35);
}

QLineEdit:hover
{
	border: 2px solid rgb(48, 48, 48);
}
QLineEdit:focus
{
	border: 2px solid rgb(0, 180, 255);
	background-color: rgb(45, 45, 45);
}</string>
              </property>
              <property name="placeholderText">
               <string>Количество участников</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>56</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>379</width>
             <height>45</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="3">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>379</width>
             <height>44</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Buttons_bar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>207</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="createButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	border: 2px solid rgb(38, 38, 38);
	border-radius: 15px;
	font-family: 'Roboto Regular';
	font-size: 16px;
	background-color: rgb(35, 35 , 35);
}
QPushButton::hover
{
	border: 2px solid rgb(0, 115, 0);
	background-color: rgb(45, 45, 45);
}
QPushButton::pressed
{
	border: 2px solid rgb(0, 85, 0);
	background-color: rgb(35, 35 , 35);
}</string>
           </property>
           <property name="text">
            <string>Создать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	border: 2px solid rgb(38, 38, 38);
	border-radius:  15px;
	font-family: 'Roboto Regular';
	font-size: 16px;
	background-color: rgb(35, 35 , 35);
}
QPushButton::hover
{
	border: 2px solid rgb(214, 0, 0);
	background-color: rgb(45, 45, 45);
}
QPushButton::pressed
{
	border: 2px solid rgb(140, 0, 0);
	background-color: rgb(35, 35 , 35);
}</string>
           </property>
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
